nav2_util/include/nav2_util/path_utils.hpp

```diff
@@ -38,7 +38,14 @@ struct PathSearchResult
 };
 /**
  * @brief Finds the minimum distance from a robot pose to a path segment.
- *
+ *Especially considering we need to explain to users that if start index
```

Comment:
Please actually read my comment and extract the information to write a comment. Obviously we shouldn't write "Especially considering we need to explain to users...." :wink: